### ISSUE TITLE
Prints with Carthage

### ISSUE THREAD

**Issue by bitomule** [caiyue1993/IceCream#52]
## Expected behavior

See IceCream debug logs.

## Actual behavior(optional)

No logs.

## Steps to reproduce the problem(optional)

Use IceCream with Carthage.

Is this expected? I don't see any debug info and don't know if I'm doing something wrong or just the expected behavior.

**Comment by caiyue1993** [comment]
So does this only happen with Carthage? And how do you use IceCream in your project?

**Comment by bitomule** [comment]
Yep, and I've tested installing with cocoapods and there print statements work.

**Comment by bitomule** [comment]
Great! Thanks @caiyue1993
